docs/gds-supported-platform/aws-service-operator.md

```diff
@@ -0,0 +1,27 @@
+# Using AWS Service Operator
+
+AWS Service Operator is a tool we use to allow GSP users to write kubeyaml resources that will directly map to AWS resources.
+It consists of a container that runs essentially a daemon, and a (slightly modified in our case) kubeyaml config that sets up the container, provides a bunch of custom resource definitions (e.g., there is a definition in there for SQS Queues), etc. - it also gives the container access to monitor stuff going on around the cluster.
```

Comment:
Who is this documentation for? Is it for users of the cluster, or operators of the cluster?  If the former, we don't need to give that much detail about the underlying daemon in the first paragraph introducing this.  It might be worth moving down to a different section? Maybe a "how it works" section with a "You don't need to know this to use it; it's for interest only" label?